packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -167,13 +171,13 @@ def _generate_tokens(self, pkg_root_path, package_name, version):
         apiview.add_navigation(navigation)
 
         # Generate tokens
-        modules = module_dict.keys()
+        modules = self.module_dict.keys()
```

Comment:
I have to assume this fix resolved bugs. Or is this for correctness?

Reply:
module_dict needed be referred in other instance methods instead of just within this method to get list of modules parsed.